### PR TITLE
 GC-57 forceDelete as url query string

### PR DESCRIPTION
**Pull request by michalstrzelecki**
Search for id in trashed and not trashed objects to perform force delete (hard delete) on non trashed objects.

### CODE REVIEW

File: src/Gzero/Repository/BaseRepository.php [hunk 1]
```diff
@@ -63,7 +63,7 @@ public function getById($id)
      */
     public function getDeletedById($id)
     {
-        return $this->newORMQuery()->onlyTrashed()->find($id);
+        return $this->newORMQuery()->withTrashed()->find($id);
```
Comment (AdrianSkierniewski) [inline src/Gzero/Repository/BaseRepository.php]: I don't understand this change, why getDeletedById is returning all records no mater if it's deleted or not? It should return only deleted records

**Comment by AdrianSkierniewski** [comment]
Second thing is that I'm not seeing any test changes, so it looks like this function isn't covered at all, we should fix that.

**Comment by michalstrzelecki** [comment]
I want to change it becouse it was failing with force deleting action for entities that were not in trash. See https://github.com/GrupaZero/api/pull/34/files, tests/functional/AdminBlockCest.php

File: tests/functional/ContentRepositoryTest.php [hunk 2]
```diff
@@ -489,6 +489,76 @@ public function it_checks_if_parent_is_proper_type()
         );
     }
 
+    /**
+     * @test
+     */
+    public function it_should_retrive_id_of_non_trashed_content() {
```
Comment (PSkierniewski) [inline tests/functional/ContentRepositoryTest.php]: This metod returns whole content, not only it's id, so an more accurate name will be "it_should_retrive_non_trashed_content", same for blocks tests.

**Pull request merged**